infrastructure/proto/log_replication_metadata.proto

```diff
@@ -129,6 +129,7 @@ message ReplicationEvent {
 
   enum ReplicationEventType {
     FORCE_SNAPSHOT_SYNC = 0;
+    UPGRADE_COMPLETION_FORCE_SNAPSHOT_SYNC = 1;
```

Comment:
I thought just naming it as `UPGRADE_COMPLETION` or `ROLLING_UPGRADE_COMPLETION` could be better?

Reply:
I agree it would be better to change to `ROLLING_UPGRADE_COMPLETION` if we handled other upgrade completion tasks as well on this event, but for now since it's just the intent to queue a snapshot sync for all sessions maybe keeping it specific is better.

Reply:
@shreayp could you please add this as a comment? By reading the event name it is not clear the intent to queue a snapshot sync